### PR TITLE
#1914 Added my name to credits

### LINKED ISSUE #1914: VisishtKamalapuram-Credits
### Description

Add my name to the credits page 

### Acceptance Criteria

See my name on the credits page 

### Proposed Solution

Add my name to the CreditsPage.tsx file 

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by vishyka**
## Changes

Added name to Credits

## Notes
-

## Screenshots
Screenshot 2024-01-23 at 11 04 43 PM

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #1914

### CODE REVIEW

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -193,7 +193,8 @@ const CreditsPage: React.FC = () => {
     { name: 'Rup Jaisinghani', color: '#065535' },
     { name: 'Jack Dreifus', color: '#014421' },
     { name: 'Vinay Pillai', color: '#42458e' },
-    { name: 'Will Anderson', color: '#ebb145' }
+    { name: 'Will Anderson', color: '#ebb145' },
+    { name: 'Visisht Kamalapuram', color: '#3083AA'}
```
Comment (Aaryan1203) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:197]: Your prettier test is failing which means that your code is not formatted correctly. To fix it you can add a space right before the closing bracket

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
We also don't want to push up the yarn lock file. The easiest fix can be to copy over your previous changes from this file and then push it up again.

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx (hunk 1 above)
Comment (vishyka) [reply src/frontend/src/pages/CreditsPage/CreditsPage.tsx]: I fixed the formatting for the code, but I am not sure how to fix the yarn.lock file. Should I just replace new yarn.lock file on my branch with the lines from the old one on master? Sorry and thank you

File: yarn.lock [hunk 2]
```diff
@@ -6164,9 +6164,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.0, caniuse-lite@npm:^1.0.30000981, caniuse-lite@npm:^1.0.30001109, caniuse-lite@npm:^1.0.30001125, caniuse-lite@npm:^1.0.30001503":
-  version: 1.0.30001513
```
Comment (RChandler234) [inline yarn.lock:6167]: you should revert the changes to this file by running: `git checkout develop -- yarn.lock`

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — DISMISSED** [review]
LGTM 🔥
